config/schedule.rb

```diff
@@ -22,3 +22,7 @@
 every :day, at: '1am' do
   rake 'dump:full', environment: 'production'
 end
+
+every :day do
```

Comment:
I was thinking, updating all of the papers at the same time every month might be a bit heavy on the load. By running this once a day, we update all the papers that haven't been for 31 days, more or less evenly throughout the month.